src/schema/v2/schema.ts

```diff
@@ -235,6 +237,12 @@ const OptionalFieldDirective = new GraphQLDirective({
   locations: [DirectiveLocation.FIELD],
 })
 
+const marketingCollectionUnstitchedRootField = config.USE_UNSTITCHED_MARKETING_COLLECTION_SCHEMA
```

Comment:
This approach will get used based on the flag inside of https://github.com/artsy/metaphysics/blob/6fb7293eff13b17e1472150be114b0c69c899cb3/src/lib/stitching/gravity/v2/stitching.ts#L77 as well (for example) - conditionally include that schema stitching snippet, or not.